data/json/items/gun/45.json

```diff
@@ -367,10 +371,10 @@
     "color": "brown",
     "ammo": [ "45" ],
     "skill": "smg",
-    "ranged_damage": { "damage_type": "bullet", "amount": 2 },
     "dispersion": 360,
     "durability": 7,
     "min_cycle_recoil": 540,
+    "default_mods": [ "muzzle_brake_mod" ],
```

Comment:
The tommy gun photo on wikipedia doesn't look like it has a brake, is this sourced from elsewhere?

Reply:
ingame M1928A1 doesn't have it, so i thinking about "chekup" on weapon mods later. It's not a single case, many guns have integrated muzzle brakes, some have "longest side" of folded gun, while having folded stock.